include/dev/dram/device.h

```diff
@@ -21,23 +21,47 @@
  *
  * mr2      Stores current value of MR2 in DRAM.
  * mr13     Stores current value of MR13 in DRAM.
+ * cal      Pointer to current DRAM freq cal structure.
+ * cfg      Pointer to current DRAM freq cfg structure.
  */
 typedef struct dram_dev_t
 {
-    uint8_t mr2;
-    uint8_t mr13;
+    uint8_t         mr2;
+    uint8_t         mr13;
+    dram_freq_cal_t *cal;
+    dram_freq_cfg_t *cfg;
 } dram_dev_t;
 
 /**
  * @brief   DRAM Initialization
  *
  * @details Initializes the DRAM device at the device level.
  *
- * @param[in]   dram    pointer to DRAM device.
+ * @param[in]   dram        pointer to DRAM device.
+ * @param[in]   dram_cfg    pointer to DRAM Frequency Configuration data.
+ * @param[in]   dram_cal    pointer to DRAM Frequency Calibration data.
  *
  * @return      void
  */
-void dram_init(dram_dev_t *dram);
+void dram_init(dram_dev_t *dram,
+               dram_freq_cfg_t *dram_cfg,
+               dram_freq_cal_t *dram_cal);
+
+/**
+ * @brief   DRAM Frequency Set
+ *
+ * @details Sets frequency specific config and calibration in DRAM devicee
+    dram->cal = dram_cal;
```

Comment:
Is this intentionally here?

Reply:
No, I think that accidentally got pasted there unintentionally, fixed.